### PR TITLE
My record box addin code

### PR DESCRIPTION
**Pull request by blackbaud-akashmehra**
My record box addin code

### CODE REVIEW

File: src/app/my-box/my-box.component.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          controls: [
+            {
+              name: 'Action 1',
+              action: 'action1',
+            },
+            {
+              name: 'Action 2',
+              action: 'action2',
+            },
+          ],
```
Comment (michael-tims) [inline src/app/my-box/my-box.component.ts:34]: You have action controls, but not handling them when they're clicked.  you can either remove them so we just don't demo the context menu, or provide a handler for them that indicates when they've been click (maybe toast messages)

Comment (blackbaud-akashmehra) [reply src/app/my-box/my-box.component.ts:37]: Added action handler, opening a toast message on action click. @michael-tims Thank you!

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "@skyux/packages": "^10.0.0",
     "@skyux/router": "^10.0.0",
     "@skyux/theme": "^10.0.0",
+    "@skyux/toast": "^10.44.2",
```
Comment (michael-tims) [inline package.json:33]: Let's put this package at `"@skyux/toast": "^10.0.0",` to match the others.

File: src/app/my-box/my-box.component.ts (hunk 1 above)
Comment (michael-tims) [reply src/app/my-box/my-box.component.ts:37]: Thanks!

File: package.json (hunk 2 above)
Comment (blackbaud-akashmehra) [reply package.json]: Done

**Review by michael-tims — APPROVED** [review]

**Pull request merged**